Ensure true system changes are reported as such

dagwieers opened the pull request:
The current playbook reports changes for tasks that do not change
anything to the system (e.g. stat). This change ensures only real
changes are reported as changed.

dagwieers committed:
Ensure true system changes are reported as such

The current playbook reports changes for tasks that do not change
anything to the system (e.g. stat). This change ensures only real
changes are reported as changed.

inkatze commented:
Thanks for the PRs! Learned a couple new things 👍 sorry it took me so long to review all of them.

dagwieers commented:
I implemented my own role now, because we are deploying RH SSO which includes both JBoss as well as Keycloak in a single archive. I had a few more modifications, and modified it using the new YAML syntax.
